bin/local-env/install-wordpress.sh

```diff
@@ -129,12 +129,16 @@ echo -e $(status_message "Activating Web Stories plugin...")
 wp plugin activate web-stories --quiet
 
 # Install & activate testing plugins.
+
+# Only install gutenberg on latest version of WordPress. 
+if [ "$WP_VERSION" == "latest" ]; then
```

Comment:
You mentioned in the comment that Gutenberg requires 5.6 -- should this specify installing on any version starting from 5.6?

Reply:
Or since we're testing only 5.5 and the latest it doesn't matter, I suppose?

Reply:
The most careful thing is only install on latest. We can always change later.